imports/plugins/included/product-detail-simple/client/components/variantList.js

```diff
@@ -120,6 +127,24 @@ class VariantList extends Component {
     return null;
   }
 
+  showIfNotDigital() {
+    if (this.props.isDigital === "" || !this.props.isDigital) {
```

Comment:
Use only –> `!this.props.isDigital` instead. This is because `!this.props.isDigital` will always produce `false` if `this.props.Digital` is `undefined`, `null` or an `empty string`.

Reply:
ok on it